ffmpeg/nvidia_test.go

```diff
@@ -764,4 +764,11 @@ func TestNvidia_API_AlternatingTimestamps(t *testing.T) {
 	tc.StopTranscoder()
 }
 
+func TestNvidia_ShortSegments(t *testing.T) {
+	shortSegments(t, Nvidia, 1)
+	shortSegments(t, Nvidia, 2)
+	shortSegments(t, Nvidia, 3)
+	shortSegments(t, Nvidia, 5)
```

Comment:
The pipeline is ~5 or 6 frames deep, isn't it? Maybe also add 6 and 10 frames just to sanity check and ensure the results hold? (If the results don't hold, then maybe just make a note of it in the code and we can move on)

Reply:
Done, results still hold!